plugins/removeRasterImages.js

```diff
@@ -21,7 +21,7 @@ exports.fn = () => {
         if (
           node.name === 'image' &&
           node.attributes['xlink:href'] != null &&
-          /(\.|image\/)(jpg|png|gif)/.test(node.attributes['xlink:href'])
+          /(\.|image\/)(jpg|jpeg|png|gif)/.test(node.attributes['xlink:href'])
```

Comment:
```suggestion
          /(\.|image\/)(jpe?g|png|gif)/.test(node.attributes['xlink:href'])
```

Very minor suggestion, but figured it just keeps things tidier.
Thanks for spotting this, will happily approve after.